config/dictionaries/resource.json

```diff
@@ -288,14 +313,34 @@
       {
         "id": "move-device",
         "summary": "Move device to another application",
+        "description": "",
         "method": "PATCH",
         "endpoint": "/v5/device(<ID>)",
         "filters": "",
         "data": "{\n    \"belongs_to__application\": \"<APP ID>\"\n}"
       },
+      {
+        "id": "get-target-release",
+        "summary": "Get the release device is pinned to",
+        "description": "Note: should_be_running__release will output unset or null if the device isn't pinned to any release. For an unpinned device, the device.is_on__commit field may be used to obtain the device's current state, while the application.commit field represents the device's target state.",
+        "method": "GET",
+        "endpoint": "/v5/device(<ID>)",
+        "filters": "?\\$select=should_be_running__release",
+        "data": ""
+      },
+      {
+        "id": "get-current-release",
+        "summary": "Get current release running on device",
+        "description": "The is_on__commit field fetches the hash of the current release that the device is running on",
+        "method": "GET",
+        "endpoint": "/v5/device(<ID>)",
+        "filters": "?\\$select=is_on__commit",
+        "data": ""
+      },
```

Comment:
Need more eyes on this change folks, let us know if this is accurate :eyeglasses: